packages/zoltan2/src/algorithms/order/Zoltan2_AlgAMD.hpp

```diff
@@ -146,6 +149,10 @@ class AlgAMD : public Algorithm<Adapter>
       // wgts are ignored in AMD
       model->getEdgeList(edgeIds, offsets, wgts);
 
+#if AMD_WITH_LNO_T
+    {
+
+      // SR: We will always call AMD with lno_t
```

Comment:
Answering my one question. You are right, this won't work. It will result in a compile error. I will go with the AMD_WITH_GNO_T as the result. Thanks !